### PR TITLE
Prepare deployment trends graph for module page

### PR DESCRIPTION
**Pull request by whopper**
This commit adds the needed front and back end code to create
the deployment trends graph on the module page. This includes
changes to the backend in terms of data being sent to the template.

### CODE REVIEW

File: puppetanalytics/templates/module.html [hunk 1]
```diff
@@ -1,37 +1,65 @@
...
-        ['data1', 30, 200, 100, 400, 150, 250],
-        ['data2', 50, 20, 10, 40, 15, 25]
-      ]
+<script>
+  $(document).ready(function() {
+    var yData = [{% for y in ys %}
+                  {{ y }},
+               {% endfor %}]
+
+    var xData = []
```
Comment (whopper) [inline puppetanalytics/templates/module.html]: We need to extract the jinja from the javascript here, and then make this less gross

**Comment by whopper** [comment]
Don't merge yet - see comments

**Comment by whopper** [comment]
Alright - the terrible javascript is "fixed" though there's still a better way to deal with data

**Comment by nibalizer** [comment]
Superceded by #34

**Pull request closed without merging**